### ISSUE TITLE
Incorrect display in mobile browsers

### ISSUE THREAD

**Issue by Morbids** [jackocnr/intl-tel-input#1480]
In mobile browsers, the <div class="iti iti--container"></div> container is not bound to the phone field, probably it should be bound to <div class="iti__flag-container"></div>
Or it should be different, maybe you need to edit the styles.
Attached a video

https://github.com/jackocnr/intl-tel-input/assets/516100/ccc29e7f-0494-466d-9df1-54a7504720f7

**Comment by jackocnr** [comment]
Good catch. The fullscreen popup view assumes that there are enough countries to fill the screen, and so looks bad when there are only a few countries in the list. I think a quick fix would be to just vertically center the popup. But maybe we should also add a dark grey layer covering the screen behind it, to make it clearer. I don't have time to implement this myself right now, but would be open to a pull request.

**Comment by dmar18** [comment]
G

**Comment by milkmidi** [comment]
add this style, it's look good.
```
  <style>
    .iti-mobile .iti__country-list {
      top: 50%;
      transform: translateY(-50%);
      background: white;
    }
    .iti-mobile .iti--container::before {
      content: '';
      position: absolute;
      z-index: -1;
      top: -30px;
      left: -30px;
      right: -30px;
      bottom: -30px;
      pointer-events: none;
      backdrop-filter: blur(4px);
      background-color: rgba(0, 0, 0, 0.25);
    }
  </style>
```

**Comment by SpareAymenSbei** [comment]
@milkmidi Unfortunately  that won't solve the problem
Any update regarding that issue? the issue appears on physical devices not on browser.

**Comment by MohamedAymenSbei** [comment]
@milkmidi  Unfortunately that won't solve the problem
Any update regarding that issue? the issue appears on physical devices not on browser.

**Comment by jackocnr** [comment]
Fix released in v18.3.5
